ReleaseNotes-2.11

```diff
@@ -31,6 +31,8 @@ Generic:
    of once every hour. As configuration files for Thinking Sphinx were changed, running
    a 'ts:rt:rebuild' task is needed after deployment, as user wwwrun:
    'chroot --userspec=wwwrun:www / /bin/bash -c "cd /srv/www/obs/api/ && /usr/bin/bundle.ruby2.5 exec rails ts:rt:rebuild RAILS_ENV=production"'
+ * Add the 'limit_for_search_results' configuration variable, to limit the results
```

Comment:
```suggestion
 * Add the 'limit_for_search_results' configuration variable, to allow limiting the results
```

Reply:
@eduardoj maybe like this since its not turned on by default right? so making clear its optional

Reply:
@krauselukas, agree. Changed.